plugins/aws/bedrock/modelInvocationLoggingEnabled.js

```diff
@@ -0,0 +1,45 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Bedrock Model Invocation Logging Enabled',
+    category: 'BedRock',
+    domain: 'Machine Learning',
+    description: 'Ensure that Amazon Bedrock model invocation logging is enabled.',
+    more_info: 'With invocation logging enabled, you can collect the full request data, response data, and metadata associated with all calls performed in account. This detailed logging provides valuable insights into model usage patterns, helps in troubleshooting, and enhances security by allowing for thorough analysis of model interactions. It also facilitates compliance with auditing requirements, offering a comprehensive record of model invocations.',
+    recommended_action: 'Enable invocation logging for Amazon Bedrock models.',
+    link: 'https://docs.aws.amazon.com/bedrock/latest/userguide/settings.html#model-invocation-logging',
+    apis: ['Bedrock:getModelInvocationLoggingConfiguration'],
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+        var regions = helpers.regions(settings);
+
+        async.each(regions.bedrock, function(region, rcb){
+            var invocationLoggingConfiguration = helpers.addSource(cache, source,
+                ['bedrock', 'getModelInvocationLoggingConfiguration', region]);
+
+            if (!invocationLoggingConfiguration) return rcb();
+
+            if (invocationLoggingConfiguration.err && invocationLoggingConfiguration.err.message.includes('This service may not be available in')) {
+                helpers.addResult(results, 0, 'Bedrock service is not available in this region', region);
+                return rcb();
```

Comment:
```suggestion
            if (invocationLoggingConfiguration.err && invocationLoggingConfiguration.err.message.includes('This service may not be available in')) {
                helpers.addResult(results, 0, 'Bedrock service is not available in this region', region);
                return rcb();
```
is it still needed even with updated regions?